src/lib/react-server-route.js

```diff
@@ -12,9 +13,8 @@ var HTML = require(`../pages/encrypt/index.js`);
 var routes = require('../routes.js');
 
 module.exports = function(req, res, next) {
-  var location = req.url;
-  var search = req.search || "";
-  var firstPath = location.split("/")[1];
```

Comment:
firstPath is now handled better inside locationParser, so we don't worry about split or [1] here.